aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -73,7 +73,9 @@ struct is_floating_point:
       std::is_same<T, at::Half>::value ||
       std::is_same<T, at::BFloat16>::value ||
       std::is_same<T, at::Float8_e5m2>::value ||
-      std::is_same<T, at::Float8_e4m3fn>::value> {
+      std::is_same<T, at::Float8_e5m2fnuz>::value ||
+      std::is_same<T, at::Float8_e4m3fn>::value ||
+      std::is_same<T, at::Float8_e4m3fnuz>::value> {
```

Comment:
I think it's safe to remove all float8 dtypes from here for now, as there shouln't be any vec256 instantiations (I can do it in a separate PR though)